### PR TITLE
Added more replies, changed ask again button

### LINKED ISSUE #28: Add more replies
The Magic 8 Ball sounds a tiny bit repetitive. Adding more responses would make it more enjoyable

### PR DESCRIPTION
**Pull request by kdwk**
Fixes #28, also improved clarity of in-app 'ask again' button by using 'view-refresh' symbolic icon instead of 'dialog-question-symbolic'

### CODE REVIEW

File: src/Content.vala [hunk 1]
```diff
@@ -62,10 +62,22 @@ public class ContentStack : Gtk.Stack {
             title = _("Yes."),
             result = "positive"
         },
+        Content () {
+            title = _("Just do it."),
```
Comment (cassidyjames) [inline src/Content.vala:66]: This doesn't really work; the user is not necessarily always asking if they should do something.

"Does Foobar love me?" "Just do it." is nonsensical. :D

File: src/Content.vala [hunk 2]
```diff
@@ -86,6 +98,22 @@ public class ContentStack : Gtk.Stack {
...
+        Content () {
+            title = _("Impossible to see, the future is."),
+            result = "neutral"
+        },
+        Content () {
+            title = _("Hmm..."),
+            result = "neutral"
+        },
+        Content () {
+            title = _("Always in motion the future is."),
```
Comment (cassidyjames) [inline src/Content.vala:114]: Ooh, I like this one!

File: src/Content.vala (hunk 1 above)
Comment (kdwk) [reply src/Content.vala:66]: Ah ok I'll get rid of this

**Comment by cassidyjames** [comment]
Thanks for the idea! I've added a couple of these (and others) to #36.

**Pull request closed without merging**